pkg/interface/config/webpack.dev.js

```diff
@@ -82,7 +82,7 @@ if(urbitrc.URL) {
 }
 
 module.exports = {
```

Comment:
Are we always running 'production' in dev now?

Reply:
-_-; I'll hotfix